nvme-metrics.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
```

Comment:
The script should not carry the file extension. See Debian Policy Manual section 10.4 for a reasoning: https://www.debian.org/doc/debian-policy/ch-files.html#scripts